src/main/java/edu/kis/powp/jobs2d/drivers/DriverChangeObserver.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+
+public class DriverChangeObserver implements Subscriber {
```

Comment:
The name of the observer should explain its behavior: i.e.  DriverFeature.updateDriverInfo();